api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -74,14 +87,43 @@ private Timestamp getCurrentTimestamp() {
   /** Makes a configurable number of project creation attempts. */
   public void bufferBillingProjects() {
 
-    monitoringService.sendSignal(
-        GaugeSignals.BILLING_BUFFER_AVAILABLE_PROJECTS, getCurrentBufferSize());
+    updateBillingProjectBufferMetrics();
     int creationAttempts = this.workbenchConfigProvider.get().billing.bufferRefillProjectsPerTask;
     for (int i = 0; i < creationAttempts; i++) {
       bufferBillingProject();
     }
   }
 
+  // TODO(jaycarlton): Move most (all) of this to its own cron method on a controller dedicated to
+  // polling gauge metric values.
+  private void updateBillingProjectBufferMetrics() {
+    ImmutableMap.Builder<OpenCensusStatsViewInfo, Number> signalToValueBuilder =
+        ImmutableMap.builder();
+    signalToValueBuilder.put(MonitoringViews.BILLING_BUFFER_SIZE, getCurrentBufferSize());
+    signalToValueBuilder.put(MonitoringViews.DEBUG_MILLISECONDS_SINCE_EPOCH, clock.millis());
+    signalToValueBuilder.put(MonitoringViews.DEBUG_RANDOM_DOUBLE, random.nextDouble());
```

Comment:
what is this for?

Reply:
They're just toy timeseries so we can experiment even with low traffic pre-launch (or on non-production environments). Metrics backends tend to flake when there are very low-traffic signals. I want a couple of curves that always have something to say (particularly in nearly dead environments).

Also, I didn't have a double-valued time series yet. I will soon, but this lets us exercise both sides of that coin right away.

Reply:
Do you mind throwing a comment to that effect above it? Just so we have it documented here?